File: solutions/2026-jan/5/tire_pressure.py
PSI = 14.5038


def tire_status(pressures_psi, range_bar):
    ans = []
    for pressure in pressures_psi:
        if pressure / PSI < range_bar[0]:
            ans.append("Low")
        elif pressure / PSI > range_bar[1]:
            ans.append("High")
        else:
            ans.append("Good")
    return ans

File: solutions/2026-jan/5/tire_pressure.js
const PSI = 14.5038;

function tireStatus(pressuresPSI, rangeBar) {
    const ans = [];
    for (const pressure of pressuresPSI) {
        if (pressure / PSI < rangeBar[0]) ans.push("Low");
        else if (pressure / PSI > rangeBar[1]) ans.push("High");
        else ans.push("Good");
    }
    return ans;
}
